packages/web-components/src/appointment-planner-component/server/core/index.ts

```diff
@@ -1,21 +1,41 @@
 import 'isomorphic-fetch'
-import express, { Request, Response } from 'express'
+import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import routers from './routers'
 import { Context, APIGatewayProxyEvent } from 'aws-lambda'
 import serverless from 'serverless-http'
 import { errorHandler } from '../../../common/utils/error-handler'
+import { traceIdMiddleware, createParseLog, AppResponse, AppRequest } from '@reapit/node-utils'
+import morgan from 'morgan'
+
+import * as Sentry from '@sentry/node'
+import { logger } from './logger'
+
+const parseLog = createParseLog(logger)
+const morganLogging = morgan(parseLog)
+
+if (process.env.NODE_ENV !== 'development') {
```

Comment:
This one should move to index.ts instead of app.ts. Because app.ts only run in the localhost